tests/src/GC/Stress/stress_run_readme.txt

```diff
@@ -18,7 +18,11 @@ The stress framework is built from <REPO_ROOT>\tests\src\GC\Stress\Framework
 
 The tests are built from <REPO_ROOT>\tests\src\GC\Stress\Tests
 
-The config is at <REPO_ROOT>NDP\clr\tests\src\GC\Stress\testmix_gc.config, this will be copied to the output folder of Framework
+The config is at <REPO_ROOT>\tests\src\GC\Stress\testmix_gc.config, this will be copied to the output folder of Framework
+
+The easiest way to build the Framework+Tests is by building all tests - "<REPO_ROOT>\build_test[.bat|.sh]"
```

Comment:
the easiest way to me to build the stress framework is just by doing msbuild on the .csproj file so you only build this one thing which is always easier than building a ton of stuff. on Linux you'd need to do "dotnet msbuild" which is probably very similar to the "dotnet build" command you mentioned below but I have not used that command so I'm not sure if it does anything other than just invoking msbuild. 

however, we should just make this by default build so one does not need to build it individually. I believe this was part of your plan, correct?

Reply:
Yes. Building the `.csproj` should also work. But you need to make sure that you have the overlay folder built and populated with right binaries.
Building tests takes care of all prerequisites in one step.

Reply:
Also note that I made the test `pri-0`, but marked as `BuildOnly` - that is so it builds without any extra flags, but does not run with `pri-0` tests. 
That is convenient. Also ensures that the test is not broken by accident.

Reply:
then I don't understand - if this is indeed included when you build tests in general, you wouldn't need to build it separately 'cause it would have already been built (as with any test you'd need to build the overlay stuff). in other words, the building step in the readme should just be deleted. the only reason I specifically mentioned the build step was because it was not built by default.

Reply:
Yeah. Building directly is useful if someone modifies the framework. Otherwise it is already there once tests are built.

I will add that building directly is an option for _re_-building the test, so that there is no confusion.

Reply:
added a note about rebuilding